CMakeLists.txt

```diff
@@ -13,6 +13,14 @@ endif()
 
 # Project
 project(protobuf C CXX)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
We probably shouldn't blindly override this one if someone has set it

Reply:
Reverted